index.d.ts

```diff
@@ -0,0 +1,96 @@
+declare module "presto-client" {
+  interface IPrestoClientOptions {
+    host: string;
+    ssl?: {
+      ca?: string;
+      cert: string;
+      ciphers?: string;
+      key?: string;
+      passphrase?: string;
+      secureProtocol: string;
+      pfx?: string;
+      rejectUnauthorized?: boolean;
+    };
+    port: number;
+    source: string;
+    user: string;
+    catalog: string;
+    checkInterval: number;
+    basic_auth?: {
+      user: string;
+      password: string;
+    };
+    schema?: string;
+    enableVerboseStateCallback?: boolean;
+  }
+
+  enum PrestoClientQueryStates {
+    QUEUED = "QUEUED",
+    PLANNING = "PLANNING",
+    STARTING = "STARTING",
+    RUNNING = "RUNNING",
+    FINISHED = "FINISHED",
+    CANCELED = "CANCELED",
+    FAILED = "FAILED"
+  }
+
+  interface IPrestoClientStats {
+    processedBytes: number;
+    processedRows: number;
+    wallTimeMillis: number;
+    cpuTimeMillis: number;
+    userTimeMillis: number;
+    state: PrestoClientQueryStates;
+    scheduled: boolean;
+    nodes: number;
+    totalSplits: number;
+    queuedSplits: number;
+    runningSplits: number;
+    completedSplits: number;
+  }
+
+  // NOTE: Needs to be extended with missing items
+  enum PrestoClientPrestoTypes {
+    varchar = "varchar",
+    bigint = "bigint",
+    boolean = "boolean",
+    char = "char",
+    date = "date",
+    decimal = "decimal",
+    double = "double"
+  }
+
+  interface IPrestoClientColumnMetaData {
+    name: string;
+    type: PrestoClientPrestoTypes;
+  }
+
+  type PrestoClientColumnDatum = [string, any];
+
+  interface IPrestoClientExecuteOptions {
+    query: string;
+    catalog: string;
+    schema: string;
+    timezone?: string;
+    info?: boolean;
+    cancel?: () => boolean;
+    state?: (error: any, query_id: string, stats: IPrestoClientStats) => void;
+    columns?: (error: any, data: IPrestoClientColumnMetaData[]) => void;
```

Comment:
```suggestion
    columns?: (error: any, columns: IPrestoClientColumnMetaData[]) => void;
```
to make consistent with the name used in `data`